<launch>
	<!-- zigbee node for communication -->
	<node pkg="zigbee" type="zigbee" name="zigbee" output="screen">
		<!-- node parameters -->
		<param name="zigbee_serial_name" type="string" value="/dev/ttyUSB1"/>
		<param name="zigbee_baud_rate" type="int" value="115200"/>

		<param name="zigbee_own_ID" type="int" value="1"/>
		<param name="zigbee_target_ID_num" type="int" value="0"/>
		<param name="zigbee_target_ID0" type="int" value="0"/>
	</node>

	<!-- UAV control node -->
	<node pkg="dji_sdk_demo" type="dji_sdk_client" name="dji_sdk_client" output="screen">
	<!-- Copy from zigbee nodeï¼Œmust be same -->
		<param name="zigbee_own_ID" type="int" value="1"/>
		<param name="zigbee_target_ID_num" type="int" value="0"/>
		<param name="zigbee_target_ID0" type="int" value="0"/>
	</node>
</launch>
